cmd/kube-apiserver/app/options/options_test.go

```diff
@@ -130,6 +130,7 @@ func TestAddFlags(t *testing.T) {
 			RequestTimeout:              time.Duration(2) * time.Minute,
 			MinRequestTimeout:           1800,
 			JSONPatchMaxCopyBytes:       int64(10 * 1024 * 1024),
+			MaxResourceWriteBytes:       int64(10 * 1024 * 1024),
```

Comment:
`MaxResourceWriteBytes` is a bit confusing... sounds like the maximum number of bytes we would write

http.Server uses `MaxHeaderBytes`... would `MaxResourceBodyBytes` be clearer?